src/facebook/facebookdatatypesyncadaptor.cpp

```diff
@@ -252,6 +252,8 @@ void FacebookDataTypeSyncAdaptor::signOnResponse(const SignOn::SessionData &resp
         SOCIALD_LOG_INFO("signon response for account with id" << accountId << "contained no access token");
     }
 
+    m_graphAPI = account->value(QStringLiteral("graph_api/Host")).toString();
```

Comment:
You may, however, need to select an appropriate service, unless it's also a global-service setting?